lib/salesforce_ar_sync/jobs/base_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module SalesforceArSync
+  class BaseJob < (defined?(::ApplicationJob) ? ::ApplicationJob : ActiveJob::Base)
+    queue_as do
+      SalesforceArSync.config['JOB_QUEUE']
+    end
+
+    protected
+
+    def prepare_class(klass)
+      klass = klass.camelize.constantize if klass.is_a?(String)
+      klass
```

Comment:
I feel like you don't want to override the klass param here
```suggestion
      klass.is_a?(String) ? klass.camelize.constantize : klass
```